connector/pom.xml

```diff
@@ -20,7 +20,13 @@
         <dependency>
             <groupId>org.geysermc</groupId>
             <artifactId>common</artifactId>
-            <version>1.2.1-SNAPSHOT</version>
+            <version>1.3.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
```

Comment:
This is a module for date and time serialization??? Why do we need this? (Also a note for myself)

Reply:
Maybe it's a merge conflict? I do remember seeing that dependency somewhere

Reply:
Ah. I moved it from the common package to the connector package since I didn't need it in the common package. Not sure if it's used anywhere in Geyser